### PR TITLE
Tutorial notebook explanation

### PR DESCRIPTION
**Pull request by willschlitzer**
As discussed in #182, this is to update the doc strings for the tutorials to state they assume the use of a Python notebook, how to modify the code to use a Pythons script, and how to save figures. These lines should be removed once #529 is merged.

### CODE REVIEW

**Comment by willschlitzer** [comment]
@seisman Not sure if I messed up the fetch/force push, but I'm style seeing all of the old commits in this PR.

**Comment by willschlitzer** [comment]
@GenericMappingTools/python-maintainers Do we think the explanation lines for using a notebook should be on all gallery examples as well? I think tutorials are sufficient, but I can add it in galleries as well.

**Comment by willschlitzer** [comment]
@GenericMappingTools/python Could I get a review/feedback for this pull request? I'm hoping to get it in for the documentation on v0.3.

File: examples/tutorials/3d-perspective-image.py [hunk 1]
```diff
@@ -4,6 +4,13 @@
 
 Create 3-D perspective image or surface mesh from a grid
 using :meth:`pygmt.Figure.grdview`.
+
+This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+To see the figures while using a Python script instead, use
+``fig.show(method="external)`` to display the figure in the default PDF viewer.
+
+To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
+is the desired name and file extension for the saved figure.
```
Comment (seisman) [inline examples/tutorials/3d-perspective-image.py:13]: Perhaps looks better using a `note` directive?
image


```suggestion
.. note::

    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
    To see the figures while using a Python script instead, use
    ``fig.show(method="external)`` to display the figure in the default PDF viewer.

    To save the figure, use ``fig.savefig("figname.pdf")`` where ``"figname.pdf"``
    is the desired name and file extension for the saved figure.
```

**Comment by weiji14** [comment]
Is there a way to insert a common block of text in multiple tutorial pages? Better than copying and pasting so much text around everytime we make a change.

**Comment by willschlitzer** [comment]
> Is there a way to insert a common block of text in multiple tutorial pages? Better than copying and pasting so much text around everytime we make a change.

I think this is a good idea, but I don't think we'll be able to get it done before v0.3, so I would rather have this note in here as is, for now, rather than no note at all.

File: examples/tutorials/3d-perspective-image.py (hunk 1 above)
Comment (willschlitzer) [reply examples/tutorials/3d-perspective-image.py]: I changed this in the tutorial files in dd3ab29

**Comment by seisman** [comment]
> Is there a way to insert a common block of text in multiple tutorial pages? Better than copying and pasting so much text around everytime we make a change.

It's a good idea. I tried to explore the possible ways but didn't find one. As we will delete these notes after we finish #529, I feel that the current PR is the simplest solution.

File: examples/tutorials/3d-perspective-image.py [hunk 2]
```diff
@@ -4,6 +4,15 @@
 
 Create 3-D perspective image or surface mesh from a grid
 using :meth:`pygmt.Figure.grdview`.
+
+.. note::
+
+    This tutorial assumes the use of a Python notebook, such as IPython or Jupyter Notebook.
+    To see the figures while using a Python script instead, use
+    ``fig.show(method="external)`` to display the figure in the default PDF viewer.
```
Comment (weiji14) [inline examples/tutorials/3d-perspective-image.py:12]:
```suggestion
    ``fig.show(method="external)`` to display the figure in the default image viewer.
```
Is the default PDF or PNG?

**Review by weiji14 — COMMENTED** [review]
Ok. Since it's temporary let's go with it. But one small comment.

File: examples/tutorials/3d-perspective-image.py (hunk 2 above)
Comment (willschlitzer) [reply examples/tutorials/3d-perspective-image.py:12]: > ```suggestion
>     ``fig.show(method="external)`` to display the figure in the default image viewer.
> ```
> Is the default PDF or PNG?

The Figure.show() documentation makes it seem like it is the PDF viewer.

Comment (weiji14) [reply examples/tutorials/3d-perspective-image.py:12]: Ah ok, all good then.

**Review by weiji14 — APPROVED** [review]
Feel free to merge :)

**Pull request merged**